docs/en/observability/synthetics-command-reference.asciidoc

```diff
@@ -67,6 +67,11 @@ Throttling can also be disabled in the configuration file using
 <<synthetics-configuration-monitor,`monitor.throttling`>>.
 The value defined via the CLI will take precedence.
 
+`--no-headless`::
+Runs with the browser in headful mode.
++
+This is the same as running `--playwright-options '{"headless": false}'`.
```

Comment:
Its good to inform users that we don't support running headful on the Synthetics global infrastructure. 

This is one of the main reason why I removed this flag as it was not uniform.

Reply:
My worry is that, people start using it and not seeing results in their monitor. Not worried about CLI/PW Options, but just as a Note saying we dont support headful mode in our infrastructure.

Reply:
@vigneshshanmugam good point, I've added a note to that effect if you could do another review

Reply:
Do you think the audience for this doc will already be familiar with the terms "headless" and "headful"? If not, a link to the Playwright docs might be helpful. I'll leave it up to you!

```suggestion
This is the same as setting https://playwright.dev/docs/api/class-testoptions#test-options-headless[Playwright's `headless` option] to `false` by running `--playwright-options '{"headless": false}'`.
```